blockchain/blockchain.cpp

```diff
@@ -0,0 +1,349 @@
+/*
```

Comment:
Your code is missing proper documentation (see the [typical structure of a program](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#typical-structure-of-a-program)).